targets/targets.json

```diff
@@ -3116,16 +3116,27 @@
         "inherits": ["FAMILY_STM32"],
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M4F",
-        "supported_toolchains": ["ARM", "uARM", "GCC_ARM"],
+        "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "IAR"],
         "program_cycle_s": 2,
         "extra_labels_add": [
             "STM32F4",
             "STM32F407",
             "STM32F407xG",
-            "STM32F407VG"
+            "STM32F407VG",
+            "STM_EMAC"
         ],
-        "device_has_add": ["ANALOGOUT", "TRNG", "FLASH"],
-        "release_versions": ["2"],
+        "device_has_add": ["ANALOGOUT", "TRNG", "FLASH", "EMAC"],
+        "device_has_remove": ["LPTICKER"],
+        "macros_add": ["USB_STM_HAL"],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC (need HW patch) | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_HSE_XTAL",
+                "macro_name": "CLOCK_SOURCE"
+            }
+        },
+        "release_versions": ["2", "5"],
+        "overrides": {"lse_available": 0},
```

Comment:
This line makes use of LSI for RTC .. but still we see RTC failing.. General comment for @0xc0170 : I have seen this happening on all F4 series for the RTC .. this is something we need to investigate more on..

Reply:
If there is not yet issue reported, please do